README.md

```diff
@@ -74,6 +74,13 @@ in
  * finally, run `nix-build` to build!
  * any additional missing dependencies that `nix-build` encounters should be fetched with `nix-prefetch-url` and added to `manual-repo.nix`.
 
+#### Keeping nix files separate from the project sources
+
+In some cases, it can be preferable to keep the nix packaging
+definitions separate from the project being packaged. In that
+case, move the .nix files elsewhere, and point the `src`
+attribute to the location that contains the Scala project.
+
```

Comment:
do we need to keep the same folder structure like `$newRoot/project/repo.nix` when moving the files?